test/scenarios/bookshop/localization.test.js

```diff
@@ -13,7 +56,7 @@ describe('Bookshop - localization', () => {
     const default_ = await SELECT.localized.from('AdminService.Books', {ID: 201}).columns('ID', 'title')
     expect(default_.title).to.be.eq('Wuthering Heights')
 
-    await cds.tx({ locale: 'de' }, async (tx) => {
+    await cds.tx(de, async (tx) => {
```

Comment:
How is it possible this test worked ?

Reply:
That worked as it used the compiler-generated `localized.AdminService.Books` table, which had the correct session_context usage in.

Note: `de` = `{ locale: 'de' }`